iOS/ExportOptions-AdHoc.plist

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think this should be in the `.gitignore` as well (it looks like there's a team id there and we could potentially use it to hold other secrets).

Reply:
Agreed